java/core/src/main/java/co/worklytics/psoxy/impl/SanitizerImpl.java

```diff
@@ -107,9 +108,6 @@ public String sanitize(@NonNull URL url, @NonNull String jsonResponse) {
             compiledPseudonymizationsWithOriginals =
                 compile(options.getRules().getPseudonymizationWithOriginals());
         }
```

Comment:
blocks above are initialization, which should move into DI factory??

Reply:
No, should be moved to constructor. I'll do it.